ONSdigital/response-operations-ui
# Edit existing users

# What and why?

This PR adds the edit and delete internal user functionality to response ops.

# How to test?

- Build enviornment
- Run acceptance tests to seed data
- Add the users.admin permission to the 'uaa_user' (in public.groups find the permission, copy the uuid then create a new row in the public.group_membership table for that user for that group) to make the 'manage user accounts' link appear.
- Test out the functionality.  You can edit any user that isn't yourself.  You can add/remove permissions and you can delete the user.

# Trello
https://trello.com/c/iOwV8EaM/1404-role-based-access-edit-existing-user
